### ISSUE TITLE
SR: Settings - Security: The auto-suggestions are announced without a role

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74839]
## Action Performed
1. Using Chrome+JAWS, open the URL - https://new.expensify.com/
2. Navigate using the tab key to the Account button and press Enter to activate it.
3. Navigate using the tab key to the Profile button and press Enter to activate it.
4. Navigate to the Pronouns button and press Enter to activate it.
5. In the modal, navigate to the Pronouns Edit field and press Enter to activate it.
6. Type a few characters into the Pronouns input box to trigger the auto-suggestion list.
7. Observe the screen reader announcement for the suggestions.

Other Occurrences:
1. On Workspace - Company cards: the options for countries
2. On Workspace - Create Workspace - Default Currency
3. On Wallet - Add bank account

## Expected Result
Each item in the auto-suggestion list should have a programmatic role (such as role="option" within a listbox) to ensure that screen readers announce them correctly as selectable options. Users should clearly understand that these elements can be interacted with or chosen from the list.

## Actual Result
When navigating to the pronoun field suggestions, options such as “Call me by my name” are announced without a role. The screen reader only reads the visible text but does not identify the suggestion as a selectable option, causing confusion for users relying on assistive technology.

HTML:

<div aria-label="Call me by my name" role="presentation" data-tag="pressable" data-selection-scrapper-hidden-element="true" data-inner-box-shadow-element="false" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21" id="callMeByMyName" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style="background-color: rgb(10, 46, 37);"><div class="css-g5y9jx r-13awgt0 r-1wtj0ep r-1awozwy r-18u37iz r-b5h31w r-1ah4tor r-9ffhgg r-lrvibr r-adacv r-kzbkwu r-ttdzmv" data-testid="base-list-item-callMeByMyName" style="background-color: rgb(10, 46, 37);"><div class="css-g5y9jx r-13awgt0 r-1habvwh"><div dir="auto" class="css-146c3p1 r-dnmrzs r-1udbk01 r-1iln25a" style="color: rgb(231, 236, 233); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; min-height: 20px; line-height: 20px; white-space: pre; text-decoration-line: none; overflow: hidden;">Call me by my name</div></div></div></div>

## Area issue was found
Settings-Security

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Assign the appropriate ARIA role to auto-suggestion elements.
Eg:
<div role="option" aria-selected="false">Call me by my name</div>
Refer:
https://webaim.org/techniques/aria/

## Environment(s)
Windows Windows 11 - Chrome

<img width="1919" height="968" alt="Image" src="https://github.com/user-attachments/assets/7fb5330e-8d84-4691-a0e6-1159715d7345" />

https://github.com/user-attachments/assets/f3bb965f-6a87-412f-81ec-a59e2347ab8c

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6994708
**Issue reported by:** NA
**Slack conversation:** NA
